MovieAppPetProject/MovieAppPetProject/GetMovieDetailsByID/GetMovieDetails-ViewModel.swift

```diff
@@ -10,54 +10,48 @@ import Foundation
 class MovieDetailsViewModel {
 
     // MARK: - Variables
-    var movieDetails: MovieDetails?
-    typealias MovieDetailsCompletion = (Result<MovieDetails, CustomError>) -> Void
-    private var selectedSearchMovie: SearchMoviesResults?
-    private var selectedHomeScreenMovies: NowPlayingResults?
-    private let movieDetailsRepository: MovieDetailsRepositoryType?
+    private let movieDetailsRepository: MovieDetailsRepositoryType
+    private let movieID: Int
+    var movieDetails: MovieDetails? {
+        didSet {
```

Comment:
you shouldn't need this did set.. Something seems off if you really need this

Reply:
you should probably be using reloadTableView() or something similar then

Reply:
I have no table view on this screen, the movie details are  going to be laid out in a somewhat particular way in various sizes and places on screen

Reply:
hmm, still seems a bit weird. we can come back to this if i is working

Reply:
ok cool, will give you a shout about it when I come back to refactor it